File: Projekat_WEB2_backend/Projekat_WEB2_backend/Dto/PorudzbinaPrikazDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Projekat_WEB2_backend.Dto
{
    public class PorudzbinaPrikazDto
    {
        public long Id { get; set; }
        public string Adresa { get; set; }
        public string Komentar { get; set; }
        public DateTime DatumKreiranja { get; set; }
        public DateTime DatumDostave { get; set; }
        public double CenaPorudzbine { get; set; }
        public List<string> ImenaArtikala { get; set; }
    }
}

File: projekat_web2_frontend/src/Models/PorudzbinaPrikazDto.js
export default class PorudzbinaPrikazDto {
    constructor(data){
        this.id = data.id;
        this.adresa = data.adresa;
        this.komentar = data.komentar;
        this.datumKreiranja = data.datumKreiranja;
        this.datumDostave = data.datumDostave;
        this.cenaPorudzbine = data.cenaPorudzbine;
        this.imenaArtikala = [];
    }

    addAllImenaArtikala(imenaArtikala){
        for(var i = 0; i < imenaArtikala.length; i++){
            this.imenaArtikala.push(imenaArtikala[i]);
        }
    }
}